libcloud/compute/drivers/gce.py

```diff
@@ -131,6 +132,80 @@ def request(self, *args, **kwargs):
 
         return response
 
+    def request_aggregated_items(self, api_name):
```

Comment:
I think this always returns only up to 500 items, as the `if self.gce_params:` condition few lines above will always evaluate to false.